packages/hebao_v1/contracts/iface/SubAccount.sol

```diff
@@ -94,17 +94,19 @@ contract SubAccount
     /// @param wallet The wallet's address.
     /// @param token The token's address, use 0x0 for Ether.
     /// @param amount The amount to consider.
+    /// @param loan The loan or deposit to consider.
```

Comment:
@param isLoan True to query the loan interest; false to query the saving interest.